html/elements/input.json

```
@@ -1380,6 +1380,64 @@
               "deprecated": true
             }
           }
+        },
+        "webkitdirectory": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "7"
+              },
+              "chrome_android": [
+                {
+                  "version_added": "132"
+                },
+                {
+                  "version_added": "131",
+                  "impl_url": "https://crbug.com/376834374",
+                  "partial_implementation": true,
+                  "notes": "In Chrome for Android 131, choosing a directory crashes the browser."
```

Instead of using `impl_url` here, refer to the bug in the `notes`:

```suggestion
                  "partial_implementation": true,
                  "notes": "In Chrome for Android 131, choosing a directory crashes the browser (see [bug 376834374](https://crbug.com/376834374))."
```